scripts/collect_backref_cost_interval_search_v3_work.py

```
@@ -0,0 +1,125 @@
+    for case in manifest["cases"]:
+        source = args.corpus / case["file"]
+        if sha256(source) != case["sha256"]:
+            raise RuntimeError(f"corpus hash mismatch: {case['case_id']}")
+        for method in (4, 6):
+            pair = {}
+            for variant in ("baseline", "candidate"):
+                env = os.environ.copy()
+                env.pop(RUNTIME, None)
+                if variant == "candidate":
+                    env[RUNTIME] = "1"
+                result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)